app/services/list_volunteer_opportunities.rb

```diff
@@ -0,0 +1,59 @@
+class ListVolunteerOpportunities
+
+  def self.with(listener, organisations, feature = Feature, http_party = HTTParty)
+    new(listener, organisations, feature, http_party).send(:run)
+  end
+
+  private
+
+  attr_reader :organisations, :listener, :feature, :http_party
+
+  def initialize(listener, organisations, feature, http_party)
+    @listener = listener
+    @organisations = organisations
+    @feature = feature
+    @http_party = http_party
+  end
+
+  def run
+    harrow_markers = listener.build_map_markers(organisations)
+    return [harrow_markers, []] unless feature.active? :doit_volunteer_opportunities
+    doit_markers = listener.build_map_markers(doit_orgs, :doit, false)
+    markers = merge_json_markers(harrow_markers, doit_markers)
+    [markers, doit_orgs]
+  end
+
+	def doit_orgs
+		host = 'https://api.do-it.org'
+    href = "/v1/opportunities\?lat\=51.5978\&lng\=-0.3370\&miles\=0.5 "
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)